build.gradle

```diff
@@ -68,9 +68,7 @@ repositories {
 
 dependencies {
     api ('uk.ac.ebi.biosamples:models-core:5.+')
-
-    api 'org.apache.xmlbeans:xmlbeans:2.6.0'
-
+    api 'org.apache.xmlbeans:xmlbeans:5.2.0'
```

Comment:
the project was upgraded to JDK17 a while ago and xmlsbeans version 2.6.0 continued to worked fine. what kind of problems did you experience @dipayan1985 due to which the version had to be changed?